javaparser-core/src/main/javacc/java.jj

```diff
@@ -4597,22 +4598,30 @@ SwitchEntry SwitchEntry():
     SwitchEntry ret;
     Expression expression = null;
     Statement stmt = null;
+    boolean isDefault = false;
 }
 {
     (
         "case" { begin=token(); }
-        label = ConditionalExpression() { labels = add(labels, label); }
         (
-            ","
-            label = ConditionalExpression()  { labels = add(labels, label); }
-        )*
+            LOOKAHEAD(3)
```

Comment:
Would it be possible to organise the configuration of the parser according to the JSL? This hasn't necessarily been the case previously, but I think it would be easier to read. This can be done later if you wish.
https://docs.oracle.com/javase/specs/jls/se21/html/jls-14.html#jls-14.11.1

Reply:
I had it organised that way before, but ran into an issue where the wrong alternative was chosen. This was at the start of me working on this though, so I was still very much figuring things out. I'll rearrange them again and see if I run into the same issue and if it can be fixed with a lookahead (which I may have placed in the wrong position), because I agree that keeping it consistent with the JLS would be nicer